### PR TITLE
Migrate CORS to Ingress

### PR DESCRIPTION
**Pull request by pperzyna**
## Description

The CORS mechanism has been implemented on the infrastructure side - on ingress controller.

## Type

- [ ] Bug Fix
- [ ] Feature Addition
- [x] Refactor
- [ ] HotFix

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by whiteshadow-gr — APPROVED** [review]

**Review by dataswifty — CHANGES_REQUESTED** [review]
Please add a comment that says we have moved CORS into infra.

**Review by augustinas — COMMENTED** [review]
This PR still has a blocking review. @pperzyna @dataswifty can you resolve it? 
From my perspective I would add that any unused code should be removed rather than commented out.

File: hat/conf/play.conf [hunk 1]
```diff
@@ -86,7 +86,7 @@ play {
     enabled = [
         "com.github.stijndehaes.playprometheusfilters.filters.StatusAndRouteLatencyAndCounterFilter",
         "org.hatdex.hat.utils.LoggingFilter",
-        "play.filters.cors.CORSFilter",
+//        "play.filters.cors.CORSFilter",
```
Comment (dataswifty) [inline hat/conf/play.conf:89]: I would delete this.

Comment (pperzyna) [reply hat/conf/play.conf]: deleted

**Pull request merged**